src/api/app/controllers/webui/users/bs_requests_controller.rb

```diff
@@ -54,15 +63,8 @@ def request_method
         REQUEST_METHODS[params[:dataTableId]] || :requests
       end
 
-      # TODO: Remove this old index action when request_index feature is rolled-over
-      def index_legacy
-        parsed_params = BsRequest::DataTable::ParamsParser.new(params).parsed_params
-        requests_query = BsRequest::DataTable::FindForUserOrGroupQuery.new(@user_or_group, request_method, parsed_params)
-        @requests_data_table = BsRequest::DataTable::Table.new(requests_query, parsed_params[:draw])
-
-        respond_to do |format|
-          format.json { render 'webui/shared/bs_requests/index' }
-        end
+      def redirect_beta
```

Comment:
Naming... `redirect_beta` is kind of confusing. The code redirects if the user is **not** in beta.

Maybe `redirect_legacy` is better?